frameworks/helloworld/src/main/java/com/mesosphere/sdk/helloworld/scheduler/Main.java

```diff
@@ -22,10 +22,11 @@ public static void main(String[] args) throws Exception {
                     .apiPort(8080)
                     .addPod(DefaultPodSpec.newBuilder()
                             .count(COUNT)
+                            .type("hello")
                             .addTask(DefaultTaskSpec.newBuilder()
                                     .name("hello")
                                     .goalState(GoalState.RUNNING)
-                                    .commandSpec(DefaultCommandSpec.newBuilder()
+                                    .commandSpec(DefaultCommandSpec.newBuilder("hello")
```

Comment:
Two uses of the string `"hello"`.  Time to make s constant?  A constant named `HELLO_POD_TYPE` also might make things clearer.  I was confused by the newBuilder with an argument like that.